src/canvas-player-controls.js

```diff
@@ -71,19 +71,37 @@ class CanvasPlayerControls extends videojs.EventTarget {
   }
 
   onMoveEnd(e) {
+
+    // We want to have the same behavior in VR360 Player and standar player.
+    // in touchend we want to know if was a touch click, for a click we show the bar,
+    // otherwise continue with the mouse logic.
+    if (e.type === 'touchend' && this.touchMoveCount_ < 7) {
+
+      if (this.player.userActive() === false) {
+        this.player.userActive(true);
+        return;
+      }
+
+      this.player.userActive(false);
+      return;
+    }
+
     if (!this.shouldTogglePlay) {
       return;
     }
-    this.togglePlay();
+
+    // We want the same behavior in Desktop for VR360  and standar player
+    if(e.type == 'mouseup') {
+      this.togglePlay();
+    }
+
   }
 
   onMove(e) {
-    // its hard to tap without a touchmove, if there have been less
-    // than one, we should still toggle play
-    if (e.type === 'touchmove' && this.touchMoveCount_ < 1) {
```

Comment:
why not just increase the limit here?

Reply:
We want to check when the movement has finished in the `touchend` event if it was a "touch for a click" or it was to move the camera in the canvas.

Reply:
It sounds like this is more or less what the `shouldTogglePlay` was supposed to do. Set a flag if there was enough touchmoves to exclude it this touch interaction from being a tap. I'd be interested to know if that will work as it'll be a much smaller change and won't require toggling user activity manually.

Reply:
The problem here is we need the change since we want to have the `play/pause` logic for `desktop/mouse` and the `userActive' logic for `mobile devices/touch screen` when the screen is touched. The current behavior that we have with my changes are:

**360°/VR Player iOS/Android**
**Click/touch** - Toggle `userActive()` - Show and hide control bar. 

**360°/VR Player Desktop**
**Click/mouse** -- Toggle play/pause since you have the `userActive()` in `mouseover`

These ones are in the Standard player: 

**Standard Player iOS/Android**
**Click/touch** - Toggle `userActive()` - Show and hide control bar. 

** Standard Player Desktop**
**Click/mouse** -- Toggle play/pause since you have the `userActive()` in `mouseover`